projects/packages/jetpack-mu-wpcom/src/features/custom-css/custom-css.php

```diff
@@ -0,0 +1,1247 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+// phpcs:disable Universal.Files.SeparateFunctionsFromOO.Mixed -- TODO: Move classes to appropriately-named class files.
+// @phan-file-suppress PhanRedefinedClassReference,PhanRedefineFunction,PhanRedefineClass
+/**
+ * Alternate Custom CSS source for 4.7 compat.
+ *
+ * @since 4.4.2
+ *
+ * @package automattic/jetpack
```

Comment:
```suggestion
 * @package automattic/jetpack-mu-wpcom
```
I think this should be updated to reflect the new package it is in?